src/Core/Content/Seo/SeoUrlPlaceholderHandler.php

```diff
@@ -96,7 +98,6 @@ private function createSeoMapping(SalesChannelContext $context, array $mapping):
         $query->setParameter('pathInfo', $mapping, Connection::PARAM_STR_ARRAY);
         $query->setParameter('languageId', Uuid::fromHexToBytes($context->getContext()->getLanguageId()));
         $query->setParameter('salesChannelId', Uuid::fromHexToBytes($context->getSalesChannelId()));
-        $query->addOrderBy('seo_url.sales_channel_id');
```

Comment:
thats important. sales_channel specific seo urls have to win

Reply:
Thanks for pointing this out, I didn't pay enough attention to that sales channel condition. I'll revert it in the coming week.